src/pages/settings/Wallet/ExpensifyCardPage.tsx

```diff
@@ -116,6 +116,8 @@ function ExpensifyCardPage({
         // eslint-disable-next-line rulesdir/no-thenable-actions-in-views
         Card.revealVirtualCardDetails(revealedCardID)
             .then((value) => {
+                // TODO: Card.revealVirtualCardDetails return type doesn't include TCardDetails, forcing us to type cast it here.
+                // The return type could be rewritten like Promise<Response & TCardDetails>
```

Comment:
What does this mean? Are you still working on it?

Reply:
@fabioh8010 this is an observation and a suggestion that could be applied here if you agree 😄

Reply:
Two observations:
- `TCardDetails` looks like a type param to me because of `T` prefix, can we rename it?
- I tried your approach (`Promise<Response & TCardDetails>`) and I still had to use assertion, but go for it if you think it's possible to avoid assertion